src/core/metrics/app-metrics.ts

```diff
@@ -32,14 +38,41 @@ export class AppMetrics {
   private completed: Record<string, number> = {};
   private eventLog: string[] = [];
   private settings = new SessionSettings<'KONTUR_SQ_ALERT' | 'KONTUR_SQ_LOG'>();
+  reportTemplate: {
+    name: string;
+    value: number;
+    type: string;
```

Comment:
more spcecific maybe?
```suggestion
    type: 'SUMMARY' | 'SOMETHING_ELSE';
```